tests/testsuite/cargo_build/help/stdout.log

```diff
@@ -42,7 +42,7 @@ Compilation Options:
       --profile <PROFILE-NAME>  Build artifacts with the specified profile
   -j, --jobs <N>                Number of parallel jobs, defaults to # of CPUs.
       --keep-going              Do not abort the build as soon as there is an error
-      --target <TRIPLE>         Build for the target triple
+      --target [<TRIPLE>]       Build for the target triple
```

Comment:
I wish we could have a way to avoid this showing as optional 🥲.

(Never mind we already have this issue everywhere.)